File: src/Backend/Domain/Auth.Entity/Ids4Entity/ClientPostLogoutRedirectUri.cs
using System;
using Dapper.Contrib.Plus;

namespace Auth.Entity.Ids4Entity
{
    /// <summary>
    /// 
    /// </summary>
    [Table("client_post_logout_redirect_uris")]
    public class ClientPostLogoutRedirectUri
    {
        /// <summary>
        /// 主键
        /// </summary>
        [Key]
        public int Id { get; set; }

        /// <summary>
        /// PostLogoutRedirectUri
        /// </summary>
        public string PostLogoutRedirectUri { get; set; }

        /// <summary>
        /// ClientId
        /// </summary>
        public int ClientId { get; set; }
    }
}

File: src/FrontEnd/auth-admin/src/types/ids/client-post-logout-redirect-uri.ts
interface IClientPostLogoutRedirectUri {
  id?: number;
  postLogoutRedirectUri: string;
  clientId: number;
}

export default class ClientPostLogoutRedirectUri implements IClientPostLogoutRedirectUri {
  id?: number | undefined;
  postLogoutRedirectUri: string = '';
  clientId: number = 0;
}